.github/workflows/pull-request.yml

```
@@ -449,3 +449,148 @@
+
+  Merge-Conflict-Check:
+    name: Check for Merge Conflicts
+    runs-on: ubuntu-latest
+    if: github.actor != 'dependabot[bot]'
+    needs: [Code-Quality-Checks]
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v4
+
+      - name: Check Mergeable Status via API
+        run: |  
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2181:style:11:8: Check exit code directly with e.g. 'if ! mycmd;', not indirectly with $?

(shellcheck)